### DISCUSSION TITLE
Yt-dlp in Python script

### DISCUSSION THREAD

**Discussion by blockbabak** [community/community#102944] [Programming Help]
### Body

I have posted on all freelancer sites a request for someone familiar with yt-dlp to join me for a personal project. I didn't even put a budget at the time.

A lot of people applied but none of them had more than a superficial understanding.

I asked for a demonstration of their level of experience by asking them to download two videos of their own choice using output templates and conditional initializations to differentiate any two different properties between their chosen video and download them using two different ID tags. I suggested using filter match to decide on file size against a maximum and download the one greater than a maximum in 144p otherwise in 320p. 
I even said they are welcome to choose any two videos of their own choice and they can get the information on them in advance outside the code.
I explained where to look in the documentation but none of those who applied were able to send me anything. Not even an attempt.
I am looking for someone who can at least look up documentation and understand what they mean.
This is just the first-level prototype and therefore it needs to be readable and modular so that it can be expanded.

I have a pretty good idea of the flow chart and I have even defined how to test the code. Someone did write a code but they didn't use the above elements and it was a mishmash of API calls, youtube-dl, and yt-dlp command line calls all mixed up and looks more like a spaghetti dish. It uses too much postprocessing and is not expandable at all. 

The project is on Google Cloud but I want to test it locally first before putting it in an App Engine. What I have is just not suitable.

So anyone interested please get in touch. 

Thank you

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by hubertle43100** [comment]
If interested in a yt-dlp project on Google Cloud, seeking someone with deep understanding. _Previous applicants lacked proficiency. 

The project involves downloading videos using yt-dlp with specific conditions. Must be readable and modular. Test locally before deploying on App Engine. 

Explore `GitHub, Stack Overflow, subreddits, tech forums, LinkedIn`, and local developer groups to find a suitable collaborator. 

Communicate project details and expectations for modularity and scalability. 
### Good luck! ⭐

**Comment by blockbabak** [comment]
Thank you for the email and your specifications are accurate. 

I am not sure about your references to other forums. 

I have one in LinkedIn, I was not very successful with Reddit. They all misunderstood my purpose; considering me a scammer or a spammer. 

I included references to developers hub to demonstrate that I was not after anything for free but it was for some reason totally misunderstood.

Subreddits don’t consider themselves as developer hubs. 

Please it see the attached document that includes the brief and clarification that I found to be misunderstandings.

Can that be published on GitHub?

Best regards

Babak Kamali
